tools/tick-cluster.js

```diff
@@ -570,6 +630,7 @@ function displayMenu(logFn) {
     logFn('\tK\t\tRevive suspended or killed processes');
     logFn('\tl <count>\tSuspend processes');
     logFn('\tm <count>\tTerminate processes');
+    logFn('\tr <batch size>\tRestart processes');
```

Comment:
Yeah, you're right; I put it in logical order but the rest if alphabetic. I'll reorder.